addons/website_sale_loyalty/tests/test_website_sale_loyalty_delivery.py

```diff
@@ -87,6 +87,22 @@ def setUp(self):
             'code': 'infinite-money-glitch',
         })
 
+        self.discount_50 = self.env['loyalty.program'].create({
+            'name': "50% discount code",
+            'program_type': 'promo_code',
+            'trigger': 'with_code',
+            'applies_on': 'current',
+            'rule_ids': [Command.create({
+                'code': 'test-50pc',
```

Comment:
```suggestion
                'code': "test-50pc",
```
Nitpicking, this char can be read by the user =]

Reply:
Oh, the way I've always is approached it is double quotes if its contents could be replaced without issue (e.g. with a translation), single quotes otherwise. But you're right that a discount code is still passed to the user.

I guess that means `url` & `login` should also be double quotes.

Seems like I have to rebase anyway for the runbot, so will apply it.